src/app/globals.css

```diff
@@ -1,13 +1,29 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+:root {
+  /* Fonts */
+  --font-family: 'Roboto', sans-serif;
+
+  /* Colors */
+  --primary-color: #90a686;
+  --secondary-color: #334338;
+  --primary-text-color: #000;
+  --secondary-text-color: #fff;
+  --background-color: #e9efed;
+  --accent-color: #f9c452;
+
+  /* Sizes */
+  font-size: 100%;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Comment:
Consider setting a default font-family (`font-family: var(--font-family)`) and color.